src/test/java/Test19.java

```diff
@@ -18,23 +19,16 @@ public void deletePlaylist() throws InterruptedException {
         clickToElement(newPlaylistButton);
         WebElement inputField = driver.findElement(By.cssSelector("input[placeholder='↵ to save']"));
         inputField.sendKeys("test1");
-        Actions actions = new Actions(driver);
         actions.sendKeys(inputField, Keys.RETURN).perform();
         WebElement playList = driver.findElement(By.xpath("//a[text()='test1']"));
         actions.contextClick(playList).perform();
         WebElement deleteSongButton = driver.findElement(By.xpath("//li[contains(@data-testid, 'playlist-context-menu-delete')]"));
         clickToElement(deleteSongButton);
-        //WebElement submitDeleteButton = driver.findElement(By.cssSelector("nav button[class='ok']"));
-        //clickToElement(submitDeleteButton);
         WebElement successNotify = driver.findElement(By.cssSelector("div[class='success show']"));
         Assert.assertTrue(successNotify.isDisplayed());
         Thread.sleep(3000);
         driver.navigate().refresh();
-        Thread.sleep(5000);
```

Comment:
Please, delete `throws InterruptedException` and **Thread.sleep**

**Thread. sleep is bad!** It blocks the current thread and renders it unusable for further work.
Using Thread. sleep() frequently in an automation framework is not a good practice.

Please use the Explicit Wait command, the WebDriver is directed to wait until a certain condition occurs before proceeding with executing the code.

![image](https://github.com/Kocharian-Iana/A48-QA-Automation-Framework/assets/104761482/cf5bfaac-d769-446f-963b-d2546745aa94)